tests/data/wgs1/README

```diff
@@ -0,0 +1,14 @@
+These are fastq files for a test sample.
+The _umi files were generated by removing the `/1` and `/2/ suffixes of the headers:
+zcat R1.fq.gz | sed 's/\/[1-2]//g' | gzip > R2.modified.fq.gz
+zcat R2.fq.gz | sed 's/\/[1-2]//g' | gzip > R1.modified.fq.gz
+
+Then run umi-tools extract to get the umi-barcodes:
+`umi_tools extract --stdin R1.fq.gz --read2-in R2.fq.gz --bc-pattern "NNXNN" --stdout R1_umi.fq.gz --read2-out R2_umi.fq.gz`
```

Comment:
Did you use the R1.fq.gz here? Or did you use the R1.modified.fq.gz from the previous step?

Reply:
The modified, changing it now